README.md

```diff
@@ -104,6 +105,37 @@ Here's how to build the StableHLO repo on Linux or macOS:
 
    This runs all the tests in `stablehlo/tests/`.
 
+## Python
+
+If you'd like to build the Python bindings, you'll need to install a few
+additional dependencies.
+
+```sh
+pip install  install -r ./llvm-project/mlir/python/requirements.txt
+```
+
+If you've built MLIR & StableHLO using the script above, the Python bindings
+for MLIR may already built.
+
+After you have built the project you can import the Python bindings to begin
+by modifying your Python path variable
+
+```shell
+$ PYTHONPATH="./build/python_packages/stablehlo" python3
+Python 3.11.6 (main, Oct  8 2023, 05:06:43) [GCC 13.2.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import mlir.dialects.stablehlo
+>>> from mlir.ir import Context, Location
+>>> import mlir.dialects.arith
+```
+
+You can also build a wheel yourself using the `setup.py` file.
```

Comment:
What are the steps for "building a wheel yourself using the setup.py file" ?

We try to script as much as possible, and have CI call these scripts - keep the user workflow and the thing we test as unified as possible (note that all out linting is locally runnable scripts). Are we missing a `ci_build_wheel.sh` which takes build dir paths, does some validation to make sure we build LLVM/StableHLO properly, and calls the build scripts / setup.py?

This can be follow-up work if it makes sense.

Reply:
I uploaded a new script but it only does the wheel creation and testing.
I'm not sure the right boundary for what the script should do for building StableHLO & LLVM.

For instance `ci_build_make` doesn't itself build LLVM but has that built in a separate job in the workflow file itself.
So the CI scripts themselves are not wholly contained already.
Perhaps that's a goal to move to ? Not sure.

Reply:
Agree that can be revisited, but for now looks like you've used a consistent boundary with the other CI scripts so LGTM